eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -312,8 +334,8 @@ def write_string_list_to_file(string_list, filename):
         out_file.write('\n'.join(string_list))
 
 
-def append_nsv(nsv_list, clinvar_record):
-    nsv = clinvar_record.nsv
+def append_nsv(nsv_list, crm):
```

Comment:
Again `clinvar_record_measure` instead of `crm`, in this specific method is it particularly difficult to know what this is.